hazelcast/src/main/java/com/hazelcast/client/impl/ClusterViewListenerService.java

```diff
@@ -87,7 +89,17 @@ private void sendToListeningEndpoints(ClientMessage clientMessage) {
             ClientMessage message = clientMessage.copyWithNewCorrelationId(correlationId);
             ClientEndpoint clientEndpoint = entry.getKey();
             Connection connection = clientEndpoint.getConnection();
+            write(message, connection);
```

Comment:
Why are we also catching the exception here ? I get the reasoning of doing it on the register listener part but not here

Reply:
If we don't catch exception for one endpoint, we can not send events to other healthy endpoints